src/components/StockTable/index.jsx

```diff
@@ -0,0 +1,118 @@
+import { stockInfo } from '../../shared/stockInfo';
+import StockSymbol from '../StockSymbol';
+
+import './styles.css';
+import {
+    Table,
+    Thead,
+    Tbody,
+    Tr,
+    Th,
+    Td,
+    useColorModeValue,
+    VStack,
+    Container,
+    Text,
+    Code,
+} from '@chakra-ui/react';
+import React from 'react';
+import { useStockSymbolList } from '../../hooks/useStockList';
+
+const StockTable = () => {
+    const stocks = stockInfo;
+    const { stockDetails } = useStockSymbolList();
+    console.log('Stocks Context: ', stockDetails);
+    const colorScheme = useColorModeValue('brand.200', 'brand.800');
+
+    // Changes color of text to red or green
+    const changeTextColor = (text) => {
+        const textStr = text.toString();
+        if (textStr.includes('-')) {
+            return (
+                <Text as='b' color='red'>
+                    {textStr}
+                </Text>
+            );
+        } else {
+            return (
+                <Text as='b' color='green'>
+                    {textStr}
+                </Text>
+            );
+        }
+    };
+
+    // Displays the details for a stock row
+    const TableData = () => {
```

Comment:
This should be a separate component 
The stocks information used here can easily be fetched in the new component

Reply:
Ok.